nilearn/input_data/tests/test_nifti_spheres_masker.py

```diff
@@ -258,3 +258,35 @@ def test_nifti_spheres_masker_inverse_overlap():
 
     with pytest.raises(ValueError, match='Overlap detected'):
         noverlapping_masker.inverse_transform(inv_data)
+
+
+def test_small_radius_inverse():
+    affine = np.eye(4)
+    shape = (3, 3, 3)
+
+    data = np.random.random(shape)
+    mask = np.zeros(shape)
+    mask[1, 1, 1] = 1
+    mask[2, 2, 2] = 1
+    affine = np.eye(4) * 1.2
+    seed = (1.4, 1.4, 1.4)
+
+    masker = NiftiSpheresMasker([seed], radius=0.6,
```

Comment:
I'm still having one issue here (I still will need to dig in way more into the code though): 
This code should work with a radius = 0.1 (see the corresponding test for the forward-transform), but I seem to either mask out or not include the seed voxel for a radius <0.4.

Reply:
Do you need help on that explicitly ?